edgelet/iotedge/src/support_bundle.rs

```diff
@@ -234,6 +248,61 @@ where
         Ok(state)
     }
 
+    fn write_docker_log_to_file(mut state: BundleState<M>) -> Result<BundleState<M>, Error> {
+        state.print_verbose("Getting system logs for docker");
+        let since_time: DateTime<Utc> = DateTime::from_utc(
+            NaiveDateTime::from_timestamp(state.log_options.since().into(), 0),
+            Utc,
+        );
+        let since = since_time.format("%F %T").to_string();
+
+        #[cfg(unix)]
+        let inspect = ShellCommand::new("journalctl")
+            .arg("-a")
+            .args(&["-u", "docker"])
+            .args(&["-S", &since])
+            .arg("--no-pager")
+            .output();
+
+        /* from https://docs.microsoft.com/en-us/virtualization/windowscontainers/troubleshooting#finding-logs */
+        #[cfg(windows)]
+        let inspect = ShellCommand::new("powershell.exe")
+            .arg("-NoProfile")
+            .arg("-Command")
+            .arg(&format!(
+                r#"Get-EventLog -LogName Application -Source Docker -After "{}" |
```

Comment:
Are you sure the source is `Docker` ? The URL in your comment is about Docker CE. For IoT Edge we install our own Moby engine and its service name is `iotedge-moby`

Also this uses `Get-EventLog` while `write_edgelet_log_to_file` uses `Get-WinEvent`. Is there a reason not to use `Get-WinEvent` ?

Reply:
I tested this with moby in a vm. It collects the logs, although they aren't very useful unless docker is set to debug level logging. 

I couldn't get Get-WinEvent to pull the logs. I tried ```Get-WinEvent -FilterHashTable @{ProviderName=Docker}```, but it doesn't recognize docker as an event provider